legend-pure-code-compiled-core/src/main/resources/core/pure/milestoning/milestoning.pure

```diff
@@ -53,6 +53,21 @@ function meta::pure::milestoning::reverseMilestoningTransforms(p:AbstractPropert
                                                         ,| if($p->meta::pure::milestoning::hasGeneratedMilestoningPropertyStereotype(),|[],|$p));
 }
 
+function meta::pure::milestoning::reverseMilestoningDatePropapagations(f:FunctionExpression[1]): FunctionExpression[1]
+{
+    if(!$f.originalMilestonedPropertyParametersValues->isEmpty(),| let ve = $f.originalMilestonedPropertyParametersValues->first()->filter(f|$f->instanceOf(VariableExpression));
+                                                                   let isAutoMap = if($ve->isNotEmpty(),
```

Comment:
Do we need automap checks here? Automap should be removed at a later stage I think